modules/openapi-generator/src/main/resources/JavaSpring/apiDelegate.mustache

```diff
@@ -26,7 +26,7 @@ import jakarta.annotation.Generated;
 {{^useJakartaEe}}
 import javax.annotation.Generated;
 {{/useJakartaEe}}
-
+import static {{package}}.{{classname}}.*;
```

Comment:
Would it be possible to avoid the `*` import and import the specific types?

For consistency, all the other imports above do not rely on `*`.

Reply:
I've fixed it, I don't know if I must extract this part of the template somewhere, because it's done multiple times.